Makefile

```diff
@@ -6,14 +6,18 @@ deps:
 	go get github.com/Pallinder/go-randomdata
 	go get github.com/mitchellh/cli
 	go get github.com/golang/lint/golint
+	go get golang.org/x/net/context
+	go get google.golang.org/cloud/storage
 	go get
 
 updatedeps:
 	go get -u -v github.com/aws/aws-sdk-go
-	go get -u -v  github.com/hashicorp/consul
-	go get -u -v  github.com/Pallinder/go-randomdata
-	go get -u -v  github.com/mitchellh/cli
-	go get -u -v  github.com/golang/lint/golint
+	go get -u -v github.com/hashicorp/consul
+	go get -u -v github.com/Pallinder/go-randomdata
+	go get -u -v github.com/mitchellh/cli
+	go get -u -v github.com/golang/lint/golint
+	go get -u -v golang.org/x/net/context
```

Comment:
Hi @sean- thanks for comments, should I leave that file as it is right now and skip google storage stuff?